ocaml/jane/doc/extensions/unboxed-types/index.md

```diff
@@ -285,6 +285,51 @@ Here's the list of primitives that currently support `[@layout_poly]`:
 * `%array_safe_set`
 * `%array_unsafe_get`
 * `%array_unsafe_set`
+* `%array_size`
+
+# Arrays of unboxed elements
+
+Arrays can store elements of any layout. You can think of `array` as having been declared as:
+
+```ocaml
+type ('a : any) array = (* ... *)
+```
+
+Array elements are packed according to their width. For example, arrays of
+elements whose layout is `bits32` store two elements per word.
+
+You can use normal array syntax for constructing such an array:
+
+```ocaml
+let array = [| #2l |]
+```
+
+Array operations must be declared with `[@layout_poly]` to be usable with arrays of unboxed elements.
```

Comment:
"operations" -> "primitives"?  Might also be worth explicitly noting that the names are the same for these primitives; this might be unexpected.

Reply:
Changed. I linked back to the `[@layout_opt]` section earlier in the doc rather than re-explain it.